components/Button/AlreadyReadButton.js

```diff
@@ -14,14 +14,15 @@ export default function AlreadyReadButton({ onToggle, id, isAlreadyRead }) {
           : "add to list of books I have already read"
       }
     >
-      ✔️
+      <AlreadyReadIcon $isActive={isAlreadyRead} />
     </StyledButton>
   );
 }
-
 const StyledButton = styled.button`
-  background-color: ${(props) =>
-    props.$isAlreadyRead ? "darkseagreen" : "seashell"};
-  padding: 0.3rem 0.5rem;
-  border-radius: 5px;
+  background-color: ${({ $isAlreadyRead }) =>
+    $isAlreadyRead ? "var(--color-green)" : "var(--color-light-yellow)"};
+  border-radius: 4px;
+  border: none;
+  width: 32px;
+  height: 32px;
```

Comment:
See my comment above ✨

```suggestion
  width: 2rem;
  height: 2rem;
```